### DISCUSSION TITLE
安装字体后使用未修改源文件还是无法复现出所给例子pdf的效果

### DISCUSSION THREAD

**Discussion by maihm6** [Keldos-Li/typora-latex-theme#117] [Q&A]
安装字体后使用未修改源文件还是无法复现出所给例子pdf的效果，如附件pdf所示[https://github.com/maihm6/all/blob/main/essay-template1.pdf](url)。

**Comment by maihm6** [comment]
[pdf](https://github.com/maihm6/all/blob/main/essay-template1.pdf)

**Reply by maihm6** [reply]
pdf在这里

**Comment by Keldos-Li** [comment]
抱歉，这份sample没有为多平台做适配，请您再下载部分这里（macOS）的字体：https://github.com/Keldos-Li/typora-latex-theme-fonts/tree/main/macOS

**Reply by maihm6** [reply]
感谢！我以为下载全平台里面的字体就可以了

**Reply by maihm6** [reply]
为什么安装mac平台字体后字体反而变了呢，一开始看起来只是字体小了点。导出的具体效果如[PDF](https://github.com/maihm6/all/blob/main/essay-template2.pdf)所示。

**Reply by Keldos-Li** [reply]
怎么正文变成楷体了，确实奇怪……
